airbyte-webapp-e2e-tests/cypress/pages/createConnectorPage.ts

```diff
@@ -50,5 +50,13 @@ export const enterSchema = (value: string) => {
   if (!value) {
     return;
   }
-  cy.get(schemaInput).clear().clear().type("{selectAll}{del}").type(value);
+  removeSchema();
```

Comment:
Make "enterSchema" function atomic - don't need to remove schema each time if we want to add new schema

Reply:
fixed, thanks!